src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -1,7 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
+  <PropertyGroup>
+    <IsWindows Condition="$([MSBuild]::IsOSPlatform('Windows'))">true</IsWindows>
+    <IsOSX Condition="$([MSBuild]::IsOSPlatform('OSX'))">true</IsOSX>
+    <IsLinux Condition="$([MSBuild]::IsOSPlatform('Linux'))">true</IsLinux>
+  </PropertyGroup>
```

Comment:
I find having a bunch of `Is` properties confusing, why not:

    <PropertyGroup>
      <Platform Condition="$([MSBuild]::IsOSPlatform('Windows'))">Windows</IsWindows>
      <Platform Condition="$([MSBuild]::IsOSPlatform('OSX'))">OSX</IsOSX>
      <Platform Condition="$([MSBuild]::IsOSPlatform('Linux'))">Linux</IsLinux>
    </PropertyGroup>

Just a thought... what you have obviously works.

Reply:
One thing that really confused me about the family of `Is` properties is the use of the value `true`.

It *looks* like a boolean, but really is just a string (right?).

Reply:
Yes it is actually a string in this case. I was just basically following the pattern set in https://github.com/DmitryLukyanov/mongo-csharp-driver/pull/152/files